lib/application/pages/home/collections/update/update_collection_memos.dart

```diff
@@ -1,23 +1,154 @@
 import 'package:flutter/material.dart';
```

Comment:
I'm really missing docs on these

Reply:
I tried to put some more docs but I'm struggling about which docs would be useful here. I'm a little biassed by the page development, so everything looks obvious to me in this page

Reply:
I'll suggest some.